kolibri/plugins/coach/assets/src/views/common/QuizStatus.vue

```diff
@@ -0,0 +1,237 @@
+      <KGridItem v-if="exam.archive" class="status-label" :layout12="{ span: 8 }">
+        {{ $tr('reportVisibleToLearnersLabel') }}
+      </KGridItem>
+      <KGridItem v-if="exam.archive" class="status-label" :layout12="{ span: 4 }">
+        <KSwitch
+          name="toggle-quiz-visibility"
+          style="display:inline;"
+          :checked="exam.active"
+          :value="exam.active"
+          @change="handleToggleVisibility"
+        />
+      </KGridItem>
+      <KGridItem class="status-label">
+        {{ coachString('recipientsLabel') }}
+      </KGridItem>
+      <KGridItem>
+        <div>
+          <Recipients
+            slot="value"
+            :groupNames="groupNames"
+            :hasAssignments="exam.assignments.length > 0"
+          />
+        </div>
+      </KGridItem>
+      <KGridItem class="status-label">
+        <span>{{ coachString('avgScoreLabel') }}</span>
+        <AverageScoreTooltip />
+      </KGridItem>
+      <KGridItem>
+        <Score :value="avgScore" />
+      </KGridItem>
+      <KGridItem class="status-label">
+        {{ $tr('questionOrderLabel') }}
+      </KGridItem>
+      <KGridItem>
+        {{ orderDescriptionString }}
+      </KGridItem>
+
+    </KGrid>
+    <QuizStatusModal
+      v-if="showConfirmationModal"
+      :modalHeader="coachString('openQuizLabel')"
+      :modalDetail="coachString('openQuizModalDetail')"
+      @cancel="showConfirmationModal = false"
+      @submit="handleOpenQuiz"
+    />
+    <QuizStatusModal
+      v-if="showCancellationModal"
+      :modalHeader="coachString('closeQuizLabel')"
+      :modalDetail="coachString('closeQuizModalDetail')"
+      @cancel="showCancellationModal = false"
+      @submit="handleCloseQuiz"
+    />
+  </KPageContainer>
+
+</template>
+
+
+<script>
+
+  import { ExamResource } from 'kolibri.resources';
+  import { coachStringsMixin } from './commonCoachStrings';
+  import Score from './Score';
+  import Recipients from './Recipients';
+  import QuizStatusModal from './QuizStatusModal';
+  import StatusElapsedTime from './StatusElapsedTime';
+  import AverageScoreTooltip from './AverageScoreTooltip';
+
+  export default {
+    name: 'QuizStatus',
+    components: { Score, Recipients, QuizStatusModal, StatusElapsedTime, AverageScoreTooltip },
+    mixins: [coachStringsMixin],
+    props: {
+      groupNames: {
+        type: Array,
+        required: true,
+      },
+      exam: {
+        type: Object,
+        required: true,
+      },
+      avgScore: {
+        type: Number,
+        required: false,
+      },
+    },
+    data() {
+      return {
+        showConfirmationModal: false,
+        showCancellationModal: false,
+      };
+    },
+    computed: {
+      orderDescriptionString() {
+        return this.exam.learners_see_fixed_order
+          ? this.coachString('orderFixedLabel')
+          : this.coachString('orderRandomLabel');
+      },
+      cancelStyleOverrides() {
+        return {
+          color: '#fff',
```

Comment:
Avoid hard-coding colors. Here the correct color is probably `$themeTokens.textInverted`